datafusion/physical-expr/src/aggregate/array_agg_distinct.rs

```diff
@@ -48,12 +50,14 @@ impl DistinctArrayAgg {
         expr: Arc<dyn PhysicalExpr>,
         name: impl Into<String>,
         input_data_type: DataType,
+        is_expr_nullable: bool,
```

Comment:
should we perhaps update the docstrings here to explain `is_expr_nullable` means if the input can have nulls?